osfs/os_windows.go

```diff
@@ -0,0 +1,46 @@
+// +build windows
+
+package osfs
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"gopkg.in/src-d/go-billy.v2"
+)
+
+// Stat returns the FileInfo structure describing file.
+func (fs *OS) Stat(filename string) (billy.FileInfo, error) {
+	// TODO: remove this in Go 1.9
```

Comment:
Could you add a comment explaining briefly what this is doing and why?
I think this is one of the cases where a private comment is really needed ;)